Panic when running 'helm publish'

`helm publish` takes a chart name, but it should error instead of panic. below is a sample panic. running `v0.1.0+c600ce8`

```bash
ENG000656:~ aaronschlesinger$ helm publish
panic: runtime error: index out of range

goroutine 1 [running]:
main.main.func6(0xc82018f560)
	/home/travis/gopath/src/github.com/deis/helm/helm.go:209 +0x129
github.com/deis/helm/vendor/github.com/codegangsta/cli.Command.Run(0x773ca0, 0x7, 0x0, 0x0, 0x0, 0x0, 0x0, 0x8322a0, 0x2a, 0x8c0ce0, ...)
	/home/travis/gopath/src/github.com/deis/helm/vendor/github.com/codegangsta/cli/command.go:131 +0x1055
github.com/deis/helm/vendor/github.com/codegangsta/cli.(*App).Run(0xc82018f320, 0xc820072020, 0x2, 0x2, 0x0, 0x0)
	/home/travis/gopath/src/github.com/deis/helm/vendor/github.com/codegangsta/cli/app.go:175 +0xfd6
main.main()
	/home/travis/gopath/src/github.com/deis/helm/helm.go:313 +0x1635

goroutine 19 [chan receive]:
github.com/deis/helm/vendor/github.com/golang/glog.(*loggingT).flushDaemon(0xb02740)
	/home/travis/gopath/src/github.com/deis/helm/vendor/github.com/golang/glog/glog.go:879 +0x67
created by github.com/deis/helm/vendor/github.com/golang/glog.init.1
	/home/travis/gopath/src/github.com/deis/helm/vendor/github.com/golang/glog/glog.go:410 +0x297
```

--- a/helm.go
+++ b/helm.go
@@ -210,6 +210,7 @@ is the first stage of contributing a chart upstream.
 `,
 			ArgsUsage: "[chart-name]",
 			Action: func(c *cli.Context) {
+				minArgs(c, 1, "publish")
 				action.Publish(c.Args()[0], home(c), c.Bool("force"))
 			},
 			Flags: []cli.Flag{

